client/landing/jetpack-cloud/sections/scan/history/style.scss

```diff
@@ -0,0 +1,16 @@
+.history {
+	padding: 0 10px;
+
+	&__title {
+		text-align: left;
+		margin-left: -10px;
```

Comment:
It looks like the history title might be aligned too far left:

<img width="222" alt="image" src="https://user-images.githubusercontent.com/42627630/76120305-d4f90e80-5fb6-11ea-9ace-8876e8ac8eaf.png">

Reply:
I changed the `FormattedHeader` for an `h1` and give it a specific font-size depending on the device: mobile or desktop.